File: tests/smoke/spring-mybatis-ntier/src/main/java/com/example/order/service/OrderService.java
package com.example.order.service;

import com.example.order.dao.OrderDao;
import org.springframework.stereotype.Service;

@Service
public class OrderService {
    private final OrderDao orderDao;

    public OrderService(OrderDao orderDao) {
        this.orderDao = orderDao;
    }

    public Object findById(Long id) {
        return orderDao.findById(id);
    }

    public Object create(Object dto) {
        return orderDao.insert(dto);
    }
}

File: tests/smoke/django-drf/orders/services/order_service.py
from orders.repositories.order_repository import OrderRepository


class OrderService:
    def __init__(self) -> None:
        self.repo = OrderRepository()

    def find_by_id(self, pk):
        return self.repo.find_by_id(pk)

    def create(self, data):
        return self.repo.insert(data)
